src/zenml/integrations/label_studio/steps/label_studio_standard_steps.py

```diff
@@ -0,0 +1,234 @@
+
+    Raises:
+        StackComponentInterfaceError: If no active annotator could be found.
+    """
+    annotator = context.stack.annotator
+    if annotator and annotator._connection_available():
+        preexisting_dataset_list = [
+            dataset
+            for dataset in annotator.get_datasets()
+            if dataset.get_params()["title"] == config.dataset_name
+        ]
+        if (
+            not preexisting_dataset_list
+            and annotator
+            and annotator._connection_available()
+        ):
+            registered_dataset = annotator.register_dataset_for_annotation(
+                config
+            )
+        elif preexisting_dataset_list:
+            return cast(str, preexisting_dataset_list[0].get_params()["title"])
+        else:
+            raise StackComponentInterfaceError("No active annotator.")
+
+        return cast(str, registered_dataset.get_params()["title"])
+    else:
+        raise StackComponentInterfaceError("No active annotator.")
+
+
+@step
+def get_labeled_data(dataset_name: str, context: StepContext) -> List:
+    """Gets labeled data from the dataset.
+
+    Args:
+        dataset_name: Name of the dataset.
+        context: The StepContext.
+
+    Returns:
+        List of labeled data.
+
+    Raises:
+        StackComponentInterfaceError: If no active annotator could be found.
+    """
+    # TODO [MEDIUM]: have this check for new data *since the last time this step ran*
+    annotator = context.stack.annotator
+    if annotator and annotator._connection_available():
+        dataset = annotator.get_dataset(dataset_name=dataset_name)
+        return dataset.get_labeled_tasks()
+    else:
+        raise StackComponentInterfaceError("No active annotator.")
+
+
+@step(enable_cache=False)
+def sync_new_data_to_label_studio(
+    uri: str,
+    dataset_name: str,
+    predictions: List[Dict[str, Any]],
+    config: LabelStudioDatasetSyncConfig,
+    context: StepContext,
+) -> None:
+    """Syncs new data to Label Studio.
+
+    Args:
+        uri: The URI of the data to sync.
+        dataset_name: The name of the dataset to sync to.
+        predictions: The predictions to sync.
+        config: The config for the sync.
+        context: The StepContext.
+
+    Raises:
+        ValueError: if you are trying to sync from outside ZenML.
+        StackComponentInterfaceError: If no active annotator could be found.
+    """
+    annotator = context.stack.annotator
+    # TODO: check that annotator is connected before querying it
+    dataset = annotator.get_dataset(dataset_name=dataset_name)
+    artifact_store = context.stack.artifact_store
+    if not uri.startswith(artifact_store.path):
+        raise ValueError(
+            "ZenML only currently supports syncing data passed from other ZenML steps and via the Artifact Store."
+        )
+
+    # removes the initial backslash from the prefix attribute by slicing
+    config.prefix = urlparse(uri).path[1:]
```

Comment:
Maybe a `rstrip("\")` is safer here?

Reply:
I think I went with my way because I wasn't sure if something weird happens with Windows for these paths. Also, why did you use the backslash and not the forward slash?

Reply:
Ah sorry, it should be a slash. The comment above that line talk about a backslash and that's why I wrote it.
It should also be `lstrip` instead to remove it from the beginning. Regarding windows, this should be a remote url and not have any backslashes right?